doc/users/next_whats_new/2019-03-18-SB.rst

```diff
@@ -0,0 +1,6 @@
+*progress_callback* argument to ani.save()
+-------------------------------------------------
+
+The `ani.save()` function of FuncAnimation objects 
```

Comment:
Simpler:
> The method `.FuncAnimation.save()` gained ...

Bonus: It links 😃

Reply:
Done

Reply:
In my opinion there is a file with guidelines (such as ".Class.method()" ) missing in doc/users/next_whats_new 
This would simplify writing PRs and avoid superfluous changes like this one. 
Since you are quite familiar with the rst syntax, could you add one?

Reply:
This is the standard cross-referencing syntax of sphinx.

It‘s all explained here:
https://matplotlib.org/devdocs/devel/documenting_mpl.html#referring-to-other-code

If you have ideas how to make that more understandable, we welcome suggestions and PRs on this.

Reply:
maybe it would be nice if that link would be listed in doc/users/next_whats_new/README.rst
I wasn't aware of the webpage until your commet